project/MimaFilters.scala

```diff
@@ -5,7 +5,7 @@ object MimaFilters {
   lazy val changesFor_3_2_0: Seq[ProblemFilter] = Seq(
     // Signature change in internal instance
     exclude[IncompatibleResultTypeProblem]("monix.catnap.internal.ParallelApplicative.apply"),
-    exclude[MissingClassProblem]("monix.eval.internal.TaskGather*")
```

Comment:
Something went wrong during the merge, this line shouldn't be removed

Reply:
Yeah I just noticed the merge screwed up, just fixed it